src/renderers/webgl/WebGLCubeUVMaps.js

```diff
@@ -106,6 +106,7 @@ function WebGLCubeUVMaps( renderer ) {
 
 			cubeUVmaps.delete( texture );
 			cubemapUV.dispose();
+			pmremGenerator.dispose();
```

Comment:
If you call `dispose()` the generator it's internal resource must be created on scratch during the next usage. That is not a correct fix. We must find out why the generator adds up texture references.

Reply:
Is it possible for you to share a fiddle that demonstrate the leak in an isolated test case?

Reply:
https://jsfiddle.net/catalin_enache/rfyc9jmo/44/

Reply:
I have simplified your fiddle a bit and there are no signs that `PMREMGenerator` leaks:

https://jsfiddle.net/sdbwn6qc/

Reply:
Thank you for taking the time to look int it.
Interesting.
It seems that it leaks only when doing this inside the loop:

```javascript
renderer.renderLists.dispose();
renderer.properties.dispose();
renderer.info.reset();
```

https://jsfiddle.net/catalin_enache/ndku0pbw/6/